src/App.css

```diff
@@ -28,15 +32,26 @@ form button {
   background: white;
 }
 form button {
-  color:#e8ffe8;
+  color: #e8ffe8;
   background: #3b5441;
   cursor: pointer;
   transition: all 0.3s ease;
 }
 form button:hover {
-  background:#e7b3b3;
+  background: #e7b3b3;
   color: white;
 }
+.todo-popup-container {
+  background-color: rgb(51, 92, 90);
+  z-index: 10;
+}
+.popup-form {
+  display: flex;
+  background-color: #05454b;
+  align-items: center;
+  justify-content: space-around;
+  padding: 130px;
```

Comment:
avoid using hardcoded large paddings, try to align containers with % or find ways which will create a common css for aligning in all systems @raiab